### PR TITLE
README Updates: About The Project Subsections and Minor Revisions

### PR DESCRIPTION
**Pull request by natalialuzuriaga**
## Problem

As per @decause-gov and I's discussion, we are making another round of revisions to all tiers of the README.

## Solution

- Added Project Vision as a subsection of "About the Project" to all tiers
- Added Agency Mission & Team Mission as subsection of "About the Project" for Tier 2 projects since projects in this tier are home-run
- Removed "multiple languages" from community section
- Removed core team section in Tier 2

### CODE REVIEW

File: tier2/{{cookiecutter.project_slug}}/README.md [hunk 1]
```diff
@@ -4,8 +4,15 @@
 ## About the Project
 **{project statement}**
 
-## Core Team
-An up-to-date list of core team members can be found in [MAINTAINERS.md](MAINTAINERS.md). At this time, the project is still building the core team and defining roles and responsibilities. We are eagerly seeking individuals who would like to join the community and help us define and fill these roles.
+### Project Vision
+**{project vision}**
+
+#### Agency Mission
```
Comment (decause-gov) [inline tier2/{{cookiecutter.project_slug}}/README.md:10]: Is this hierarchy intentional? Vision part of core team, with Agency Mission and team mission under agency mission?

Perhaps we can try rearranging the order of headings based on broadness>specificness, or grouping related sections? Something like:

```
## About the Project
### Project Vision
### Project Mission (is this redundant? I know there's a difference between vision/mission statements, but I am not sure how many teams/projects have/will have them.)
## Team Mission
## Core Team
```

File: tier2/{{cookiecutter.project_slug}}/README.md [hunk 2]
```diff
@@ -4,8 +4,15 @@
...
+<!--- # TODO: Good to include since this is a home-run project -->
```
Comment (decause-gov) [inline tier2/{{cookiecutter.project_slug}}/README.md:11]: whaddya think? s/home-run/agency-led

File: tier2/{{cookiecutter.project_slug}}/README.md [hunk 3]
```diff
@@ -4,8 +4,15 @@
...
+
+#### Team Mission
+<!--- # TODO: Good to include since this is a home-run project -->
```
Comment (decause-gov) [inline tier2/{{cookiecutter.project_slug}}/README.md:14]: whaddya think? s/home-run/agency-led

File: tier3/{{cookiecutter.project_slug}}/README.md [hunk 4]
```diff
@@ -4,6 +4,9 @@
 ## About the Project
 **{project statement}**
 
+### Project Vision
```
Comment (decause-gov) [inline tier3/{{cookiecutter.project_slug}}/README.md:7]: Let's add this section commented out (recommended/optional) for projects outside of tier2?

File: tier4/{{cookiecutter.project_slug}}/README.md [hunk 5]
```diff
@@ -4,6 +4,9 @@
 ## About the Project
 **{project_statement}**
 
+### Project Vision
```
Comment (decause-gov) [inline tier4/{{cookiecutter.project_slug}}/README.md:7]: Let's add this section commented out (recommended/optional) for projects outside of tier2?

File: tier1/{{cookiecutter.project_slug}}/README.md [hunk 6]
```diff
@@ -1,11 +1,12 @@
 ## **{Project Name}**
-
 **{project description}**
 
 ## About the Project
-
 **{project statement}**
 
+### Project Vision
```
Comment (decause-gov) [inline tier1/{{cookiecutter.project_slug}}/README.md:7]: Let's add this section commented out (recommended/optional) for projects outside of tier2?

**Review by decause-gov — APPROVED** [review]
Made a few suggestions about ordering/hierarchy, and optionality of project_vision outside of Tier2 projects.

Otherwise, LGTM +1 🚢 📊

File: tier2/{{cookiecutter.project_slug}}/README.md (hunk 2 above)
Comment (natalialuzuriaga) [reply tier2/{{cookiecutter.project_slug}}/README.md:11]: Agree, this wording fits better!

File: tier2/{{cookiecutter.project_slug}}/README.md (hunk 1 above)
Comment (natalialuzuriaga) [reply tier2/{{cookiecutter.project_slug}}/README.md:10]: Order confirmed, returning Core Team section

**Review by decause-gov — APPROVED** [review]
LGTM +1 🚢 👓

**Pull request merged**